addon/components/hyper-table-v2/filtering-renderers/text.hbs

```diff
@@ -8,6 +9,8 @@
   {{#if @column.definition.filterable}}
     <HyperTableV2::FilteringRenderers::Common::Search @handler={{@handler}} @column={{@column}}
                                                       @registerResetCallback={{this.registerResetCallback}} />
+    <HyperTableV2::FilteringRenderers::Common::Existence @handler={{@handler}} @column={{@column}}
+                                                         data-control-name={{concat "hypertable__column_filtering_for_" @column.definition.key "_existence_selector"}} />
```

Comment:
In the design, `Existence` is before the `Search` :)

Reply:
noted ! changing this now !